ibmwatson.class.nut

```diff
@@ -0,0 +1,263 @@
+class IBMWatson {
+
+    static version = [1, 0, 0];
+
+    static INVALID_REQUEST_ERROR = "Error: Invalid Request";
+    static INVALID_AUTH_TOKEN_ERROR = "Error: Invalid Authentication Token";
+    static INVALID_AUTH_METHOD_ERROR = "Error: Invalid API Key or Authentication Method";
+    static MISSING_RESOURCE_ERROR = "Error: Resource does not exist";
+    static CONFLICT_ERROR = "Error: Conflict, resource already exists";
+    static UNEXPECTED_ERROR = "Error: Unexpected error";
+
+    _apiKey = null;
+    _authToken = null;
+    _baseURL = null;
+
+    /***************************************************************************************
+     * Constructor
+     * Returns: null
+     * Parameters:
+     *      apiKey : string - Watson API key (generate via Accees -> API Keys)
+     *      authToken : string - Watson Authorization Token (provided when API Key is created)
+     *      orgID : string - Organization ID
+     *      version (optional) : string - Watson API version number
+     **************************************************************************************/
+    constructor(apiKey, authToken, orgID, version = "v0002") {
+        _apiKey = apiKey;
+        _authToken = authToken;
+
+        local protocol = (orgID == "quickstart") ? "http" : "https";
```

Comment:
Can you please add a comment on this somewhere. Otherwise it looks like a backdoor :)